docs/asyncapi.yaml

```diff
@@ -52,156 +51,148 @@ components:
                   - SFI
                 description: Scheme name
                 example: SFI
-          type: 'uk.gov.pay.statement.crm'
-          source: 'ffc-pay-statement-generator'
   schemas:
     PaymentStatement:
       type: object
       required:
       - businessName
-      - sbi
       - frn
+      - sbi
       - address
-      - scheme
-      - payments
       - funding
+      - fundingTotal
+      - payments
+      - scheme
       properties:
         businessName:
           $ref: '#/components/schemas/BusinessName'
-        sbi:
-          $ref: '#/components/schemas/SBI'
-        frn:
-          $ref: '#/components/schemas/FRN'
         email:
           $ref: '#/components/schemas/Email'
+        frn:
+          $ref: '#/components/schemas/FRN'
+        sbi:
+          $ref: '#/components/schemas/SBI'
         address:
           $ref: '#/components/schemas/Address'
-        scheme:
-          $ref: '#/components/schemas/Scheme'
-        payments:
+        funding:
           type: array
           items:
-            $ref: '#/components/schemas/Payment'
-        funding:
+            - $ref: '#/components/schemas/Funding'
+            - $ref: '#/components/schemas/FundingTotal'
+        payments:
           type: array
           items:
-            $ref: '#/components/schemas/Funding'
+            $ref: '#/components/schemas/Payment'
+        scheme:
+          $ref: '#/components/schemas/Scheme'
     BusinessName:
       type: string
       description: The business name
-      example: 'FFC Ltd'
-    SBI:
-      type: number
-      description: Single Business Identifier
-      minimum: 105000000
-      maximum: 999999999
-      example: 123456789
+      example: 'Summer Days Farm'
+    Email:
+      type: string
+      description: The business email address
+      example: farm@farms.com
     FRN:
       type: number
       description: Firm Reference Number
       minimum: 1000000000
       maximum: 9999999999
       example: 1234567890
-    Email:
-      type: string
-      description: The business email address
-      example: farm@farms.com
+    SBI:
+      type: number
+      description: Single Business Identifier
+      minimum: 105000000
+      maximum: 999999999
+      example: 123456789
     Address:
       type: object
       properties:
         line1:
```

Comment:
These should all be schemas and referenced

Reply:
What should all be schemas and referenced?